packages/amplify-cli/src/initialize-env.ts

```diff
@@ -151,6 +152,14 @@ export const initializeEnv = async (
     // Generate AWS exports/configuration file
     await context.amplify.onCategoryOutputsChange(context, currentAmplifyMeta);
 
+    if (context.input.command === 'env' && context.input.subCommands[0] === 'add') {
+      const cloneFromSrcEnv = await prompter.yesOrNo('Do you want to clone values from the source environment?', false);
```

Comment:
Instead of saying "source environment", displaying the name of the source environment would make it more clear which environment we mean.

Reply:
Good idea